src/wallet/wallet.h

```diff
@@ -604,17 +612,28 @@ class COutput
     }
 };
 
+/** Parameters for one iteration of Coin Selection. */
 struct CoinSelectionParams
 {
+    /** When true, try to use the Branch and Bound algorithm before falling back to knapsack. */
```

Comment:
Since this is a patch to master, I think `use_bnb` still means "only use BnB". The try BnB first and then fall back to knapsack behavior gets introduced by #17331.
```suggestion
    /** Toggles use of the Branch and Bound coin selection algorithm instead of the knapsack solver. */
```